try
{
	_<>_print(x);
}
catch(e)
{
	_<>_print("Error");
}

"PASS";
